src/BenchmarksApps/TLS/Kestrel/Program.cs

```diff
@@ -219,4 +223,25 @@ static IPEndPoint CreateIPEndPoint(UrlPrefix urlPrefix)
     }
 
     return protocols;
+}
+
+static void LogOpenSSLVersion()
```

Comment:
Looks like this code runs no matter what OS? Should probably check you're on linux?

Reply:
added check that we are on linux or macos